packages/react-devtools-shared/src/devtools/views/Components/SelectedElement.js

```diff
@@ -532,18 +537,26 @@ function OwnerView({
     [dispatch, id],
   );
 
+  const onMouseEnter = () => highlightNativeElement(id);
+
+  const onMouseLeave = () => clearHighlightNativeElement();
```

Comment:
nit: This wrapper function isn't necessary :smile:

```suggestion
  const onMouseLeave = clearHighlightNativeElement;
```

Reply:
We need a `facepalm` reaction on GitHub. 😄

Reply:
I'd use that one all the time :)